src/components/Cards/Card.tsx

```diff
@@ -24,27 +25,32 @@ const Card: React.FC<CardProps> = ({
   restaraunt,
   isDish,
   isRest,
-  cardData
+  isPopular
 }) => {
 // console.log(cardData)
 
   return (
-    <CardContainer>
+    <CardContainer isRest={isRest}>
       {restaraunt && <CardHeader>
         <h1>{restaraunt}</h1>
         </CardHeader>}
+      {isRest && (
+        <RestImage>
+          <img src={img} alt={alt}></img>
+        </RestImage>
+      )}
       {isDish && (
         <DishImage>
           <img src={img} alt={alt}></img>
         </DishImage>
       )}
```

Comment:
This can be done in a cleaner way, maybe by using props to handle how the image looks? or maybe the whole component can be even more generic, Compound Components could be a great solution for this..

Reply:
I'm not familiar with the term "Compound Components", I'll look it up.
But for sure it can be done in a cleaner way with props.
I hope I'll refactor it soon!